src/App.tsx

```diff
@@ -1,18 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalPages = items.length;
+  const perPageOptions = [3, 5, 10, 20];
```

Comment:
move it out of the component. It's a constant and we don't need to redeclare it on each render